game-1/Azivar_Snail_race/views/4995dfdb-baa6-4f86-9a37-f7ead5aa18c5.yy

```diff
@@ -5,7 +5,9 @@
     "name": "4995dfdb-baa6-4f86-9a37-f7ead5aa18c5",
     "children": [
         "ed6a955d-5826-4f98-a450-10b414266c27",
-        "cc98d028-7bdd-4680-85f3-c87a7baa481e"
+        "cc98d028-7bdd-4680-85f3-c87a7baa481e",
+        "f418569b-3bdd-4706-a0e4-364317f54032",
```

Comment:
Too lazy to pull code and see what this is. What two children does this add to one of Azivars view?